kernel/examples/read-table-multi-threaded/src/main.rs

```diff
@@ -51,6 +51,10 @@ struct Cli {
     /// to the aws metadata server, which will fail unless you're on an ec2 instance.
     #[arg(long)]
     public: bool,
+
+    /// Limit to printing only LIMIT rows.
+    #[arg(short, long)]
+    limit: Option<usize>,
```

Comment:
out of curiosity why usize instead of u64?

Reply:
oh i suppose slice/take/etc for iterators take usize. interesting though since i feel like this communicates length but i always associate usize with indexes

Reply:
yeah. I had it as a u32, but then I'd just have to cast it below, so I figured make it a usize here